docs/backup_workstations.rst

```diff
@@ -118,16 +118,92 @@ Then, copy the contents of the device's persistent volume to the directory using
 
 .. code:: sh
 
-  rsync -a --info=progress2 --no-specials --no-devices \
-      /media/amnesia/TailsData/ /media/amnesia/Backup/admin-backup
+  sudo bash -c "rsync -a --info=progress2 --no-specials --no-devices \
+      /media/amnesia/TailsData/ /media/amnesia/Backup/admin-backup && sync"
 
-Once complete, unmount the TailsData partition.
+.. note:: Note that the first path argument ends with a ``/``, but the second 
+          path argument should not!
+
+
+Once complete, unmount the TailsData partition by clicking the Eject button 
+beside its entry in the lefthand column of the file manager. When its entry is 
+no longer shown in the lefthand column, it is save to remove the 
+*Admin Workstation* USB.
 
 Repeat these steps for every device, making a new folder on the backup device
 for each device you back up.
 
 Finally, once you have completed the steps described in this section for each
-USB drive, unmount the Backup partition and store the drive somewhere safely.
+USB drive, unmount the Backup partition by clicking its Eject button. Wait until
+the Backup USB can be safely removed, and store it somewhere safely.
+
+.. note:: After the Eject button is clicked, it may be take some time before 
+          the drive can be safely removed. Wait until its entry  is removed from
+          the lefthand column of the file manager.
+
+.. _restore_workstations:
+
+Restoring a Workstation from a Backup
+-------------------------------------
+
+To recreate a backed-up *Admin Workstation*, *Journalist Workstation*, or 
+*Secure Viewing Station* Tails USB,  you will need 
+
+- your Backup USB containing the persistent volume to be restored,
+- a fresh USB stick to be set up as the new Tails USB,  
```

Comment:
Right now you're using "fresh USB" and "new USB" interchangeably to refer to the recovery target. Terms like "fresh" and "new" can be confusing when juggling multiple USB drives (at least 3, possibly more if they're doing multiple restores), Tails cloning processes, etc.

I would reword this as follows:
> a fresh USB stick to serve as the restored *Admin Workstation*, *Journalist Workstation* or *Secure Viewing Station* USB, referred to as the "workstation USB" in this document

and then update subsequent sections, "Prepare the new workstation USB", etc.

Reply:
I refer to it as the "fresh USB stick" until Tailsified, at which stage it changes to the "new Tails USB" - "workstation USB" would be ambiguous IMO as there are other workstation USBs referred to in the doc.

How about the "blank USB stick" before formatting, the "new workstation USB" afterwards?

Reply:
> How about the "blank USB stick" before formatting, the "new workstation USB" afterwards?

:+1:, that sounds very clear to me :)

Reply:
Aaand done.

Reply:
LGTM!